src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxIPGlobalProperties.cs

```diff
@@ -0,0 +1,362 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Globalization;
+using System.IO;
+
+namespace System.Net.NetworkInformation
+{
+    internal class LinuxIPGlobalProperties : IPGlobalProperties
+    {
+        private static readonly string[] newLineSeparator = new string[] { Environment.NewLine }; // Used for string splitting
+
+        public override string DhcpScopeName
+        {
+            get
+            {
+                throw new NotImplementedException();
+            }
+        }
+
+        public override string DomainName
+        {
+            get
+            {
+                return HostInformation.DomainName;
+            }
+        }
+
+        public override string HostName
+        {
+            get
+            {
+                return HostInformation.HostName;
+            }
+        }
+
+        public override bool IsWinsProxy
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        public override NetBiosNodeType NodeType
+        {
+            get
+            {
+                // Could return NetBiosNodeType.Unknown (?)
+                throw new NotImplementedException();
+            }
+        }
+
+        public override TcpConnectionInformation[] GetActiveTcpConnections()
```

Comment:
This is a very good candidate for a Unit Test: you could extract the text processing part into a different file or even class and directly reference that in a UT.